robottelo/vm.py

```diff
@@ -372,6 +372,56 @@ def execute_foreman_scap_client(self, policy_id=None):
             raise VirtualMachineError(
                 'Failed to execute foreman_scap_client run.')
 
+    def configure_rhai_client(self, activation_key, org, rhel_distro):
+        # self.vm = VirtualMachine(distro=rhel_distro)
+        # self.vm.create()
+        # Download and Install ketello-ca rpm
+        self.install_katello_cert()
+        self.register_contenthost(activation_key, org)
+
+        # Red Hat Access Insights requires RHEL 6/7 repo and it is not
+        # possible to sync the repo during the tests, Adding repo file.
+        if rhel_distro == 'rhel67':
+            rhel_repo = conf.properties['clients.rhel6_repo']
+        if rhel_distro == 'rhel71':
+            rhel_repo = conf.properties['clients.rhel7_repo']
+        try:
+            if rhel_distro == 'rhel67':
+                insights_repo = conf.properties['insights.insights_el6repo']
+            if rhel_distro == 'rhel71':
+                insights_repo = conf.properties['insights.insights_el7repo']
+        except KeyError:
+            pass
+
+        self.configure_rhel_repo(rhel_repo)
+
+        self.run('wget -O /etc/yum.repos.d/insights.repo {0}'.
```

Comment:
If the `insights_repo` is not defined (was raised KeyError exception), this will fail.